README.md

```diff
@@ -1,10 +1,46 @@
-# Release automation for `laminas/*` packages
+# Release Automation for `laminas/*` packages
 
-Please read the [`feature/`](./feature) specification to understand how
-this tool behaves.
+This project is a [Github Action](https://github.com/features/actions) that allows
+maintainers of open-source projects that follow [SemVer](https://semver.org/spec/v2.0.0.html)
+to automate the automation of releases.
 
-TL;DR: when you close a milestone:
- * a `git tag` will be created
- * a github release will be created
- * if not already existing, a release branch will be created
- * a new pull request will be created, porting changes to the next release branch from an intermediary branch (e.g. 1.2.3-merge-up-to-1.3.x)
+## Installation
+
+To use this automation in your own repository, copy the [`example/.github`](./examples/.github)
+workflows into your own project:
+
+```sh
+cd /tmp
+git clone https://github.com/laminas/automatic-releases.git
```

Comment:
Is doing a `git clone` a bit OTT?

Could we not just do something like `wget https://github.com/laminas/automatic-releases/blob/1.0.x/examples/.github/workflows/release-on-milestone-closed.yml .github/workflows/release-on-milestone-closed.yml`?

Reply:
I kinda wanted to avoid that - adds more separate CLI commands...